server/weaverbird/backends/pandas_executor/pipeline_executor.py

```diff
@@ -34,7 +34,7 @@ def execute_pipeline(
     for index, step in enumerate(steps):
         try:
             with stopwatch:
-                df = steps_executors[step.name](
+                df = steps_executors[step.name](  # type: ignore
```

Comment:
Oh sad that we have to ignore that :'(
I tried to create a type `StepExecutor` and would have like to type every of these functions with is, but I din't find a way in python to declare the type of function directly, using `Callable`. I was counting on mypy "understanding" the compatibity of this type with the definitions.
Up to search a bit more today, but in the meantime ignoring is fine

Reply:
fixed by ad20c88